pytest_doctestplus/plugin.py

```diff
@@ -33,7 +34,11 @@ def pytest_addoption(parser):
                      "plugin")
 
     parser.addoption("--doctest-rst", action="store_true",
-                     help="enable running doctests in .rst documentation")
+                     help="DEPRECATED, use doctest-glob. Enable running doctests in .rst documentation.")
+
+    parser.addoption("--text-file-format", action="store",
+                     help=("Text file format for narrative documentation. "
+                           "Options accepted are 'txt', 'tex', and 'rst'"))
```

Comment:
Similar comment to previous one in terms of wording (need to mention glob as the recommended option)